src/lib/components/DotGovBanner/DotGovBanner.svelte

```diff
@@ -1,3 +1,10 @@
+<!--
+ This component is currently not in-use on the site (only in Storybook).
+ If / when there is a desire to include it, the LDAF logo will need to be
+   replaced with either the updated "Icon" logo or the state seal. Please
+   remove the `ldaf-flat-logo-transparent` file when you do so, as this is the
+   only place where that file is in-use.
+-->
```

Comment:
Left a note here when I was cleaning up the old logo files and realized one of them was still in-use. Figured juice wasn't currently worth the squeeze to update this since this component is only in Storybook.